docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -2440,215 +2440,106 @@ And can be defined using different criteria like:
 
-.EXAMPLE: Refuse user authentication without prior machine authentication
-[source,ini,numbered]
-----
-  [EthernetEAP]
-  filter = connection_type
-  operator = is
-  value = Ethernet-EAP
+  (&(|(sAMAccountName=%{User-Name})(sAMAccountName=%{Stripped-User-Name})(cn=%{User-Name})(cn=%{Stripped-User-Name})(sAMAccountName=%{%{Stripped-User-Name}:-%{User-Name}})))
 
-  [EAPTLS]
-  filter = radius_request
-  attribute = EAP-Type
-  operator = is
-  value = TLS
+and if you define a appended ldap filter like:
 
-  [machine]
-  filter = node_info
-  attribute = machine_account
-  operator = defined
-  value = robert
+  (|(memberOf=CN=Staff,OU=Security Groups,DC=ad,DC=acme,DC=com)(wifi=enabled))
 
-  [REJECT_NO_MACHINE_AUTH_NO_EAPTLS:EthernetEAP&!machine&!EAPTLS]
-  scope = RegisteredRole
-  role = REJECT
+Then the filter will be generated like that:
 
-  [REJECT_NO_MACHINE_AUTH:EthernetEAP&!machine]
-  scope = RegistrationRole
-  role = REJECT
-----
+  (&(|(sAMAccountName=%{User-Name})(sAMAccountName=%{Stripped-User-Name})(cn=%{User-Name})(cn=%{Stripped-User-Name})(sAMAccountName=%{%{Stripped-User-Name}:-%{User-Name}}))(|(memberOf=CN=Staff,OU=Security Groups,DC=ad,DC=acme,DC=com)(wifi=enabled)))
 
-.EXAMPLE: autoregister printers and scanners
-[source,ini,numbered]
-----
-  [printers]
-  filter = node_info.device_class
-  operator = is
-  value = Printers/Scanners
+By doing that, event you wont don't need the "Search Attributes" feature, you will be able to store the user dn in the PacketFence-UserDN attribute.
 
-  [HP_Printer]
-  filter = node_info.mac
-  operator = regex
-  value = ^(00:1e:0b:0e:22:4c|00:23:7d:91:0e:86|00:1f:29:19:1d:4f|00:18:fe:a0:26:e2|00:1e:0b:1b:d2:e7|2c:76:8a:3f:21:cd|00:14:38:d8:a1:14|00:14:38:de:b6:0a)$
 
-  [autoreg_printers:(printers)]
-  scope = AutoRegister
-  role = printers
+basedn condition
+^^^^^^^^^^^^^^^^
 
-  [printers_role:(printers)]
-  scope = NodeInfoForAutoReg
-  role = printers
-----
+This condition allow to override the default basedn in the ldap source and it will permit to test if a object is in a specific ou.
 
-You can also add notes to the device using filters. Here you can see how to add a note to all HP Printers.
 
-[source,ini,numbered]
-----
-  [Add_Note_HP_Printer:HP_Printer]
-  scope = RegisteredRole
-  role = printers
-  action = modify_node
-  action_param = mac = $mac, notes = HP PRINTER
-----
+Advanced Realm Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In PacketFence you can define multiple Realms to select on which domain you want to authenticate the users.
+
+You can define a Realm with a regex in order to match multiples format.
+
+Per example in the ACME realm we define the regex like this:
+
+  .*\.acme\.com$
+
+It mean that if you have a user coming with this username mickey@la.acme.com , FreeRADIUS will define the realm as la.acme.com (it will be included in the radius request) and PacketFence will map the user to the ACME Realm.
```

Comment:
It mean -> It means
FreeRADIUS? I think we should always say PacketFence - they don't know/care we're using FreeRADIUS.

Reply:
Realm -> realm